client/my-sites/checkout/checkout/domain-refund-policy.jsx

```diff
@@ -109,16 +111,12 @@ class DomainRefundPolicy extends React.Component {
 
 		const policies = this.getApplicablePolicies();
 
-		return (
-			<>
```

Comment:
Note to myself to test this: if we return multiple elements without being wrapped in a fragment, won't this cause a React error?

Reply:
I tested this. It doesn't seem to, but you're welcome to test it too.

Reply:
TIL [that you can return an array](https://flaviocopes.com/jsx-return-multiple-elements/). Thanks!

Reply:
Always teaching you new things through my smelly code.